use-cases.md

```diff
@@ -7,7 +7,8 @@ This section describes, how to use the Connector API in order to implement some
 * [Point of sale systems](use-cases.md#point-of-sale-systems)
 * [Guest technology systems](use-cases.md#guest-technology-systems)
 * [Reputation management systems](use-cases.md#reputation-management-systems)
-* [Mobile Key Systems](use-cases.md#mobile-key-systems)
+* [Mobile key systems](use-cases.md#mobile-key-systems)
```

Comment:
Capitalizing all significant words in a title or heading or only the 1st word is debatable.  What's our SOP?